scripts/prepare.sh

```diff
@@ -50,7 +56,13 @@ stage "Clone projects"
 clone_project https://github.com/ShunkevichDV/sc-machine.git sc-machine 0.6.0
 clone_project https://github.com/ostis-dev/sc-web.git sc-web 0.6.0
 clone_project https://github.com/ostis-dev/ims.ostis.kb.git ims.ostis.kb 432bbaa
-
+if [ $clone_monography == 1 ]; then
+	clone_project git@github.com:semsystems/monography2020.git monography2020 tex2sc
```

Comment:
```suggestion
	clone_project git@github.com:semsystems/monography2020.git ostis-standard tex2sc
```

Reply:
Move content of this "if" to separate script

Reply:
tex2sc branch should be replaced by commit hash from master branch after merge pr from monography repo